cmd/opentelemetry/app/exporter/elasticsearchexporter/spanreader/validate_query.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanreader
+
+import (
+	"errors"
+
+	"github.com/jaegertracing/jaeger/storage/spanstore"
+)
+
+var (
+	errNilQuery                   = errors.New("query is nil")
+	errServiceNameNotSet          = errors.New("service Name must be set")
```

Comment:
Not sure the error messages should start with lowercase letter when other words in the sentence use upper case. Same for messages below.